### ISSUE TITLE
MIDI.write implementation

### ISSUE THREAD

**Issue by giorrrgio** [adafruit/Adafruit_CircuitPython_USB_Host_MIDI#1]
Hi,
thank you for the library! I was wondering why there's no write implementation inside the midi class.
It's not as simple as the following, is it? 😄 
```python
def write(self, packet, num):
    self.device.write(self.out_ep, packet)
```

**Comment by tannewt** [comment]
I don't think so. There is some conversion between MIDI packets and USB MIDI container packet.

What device do you want to use it with? I don't have one for testing now.

**Comment by MacGuffin-Underscore** [comment]
I have been trying to get it to work with the Synthstrom Deluge.  I am trying to make a external screen for it, so that means sending SYSEX messages and getting back the response.  Is there a way to do this using the USB_HOST_DATA_PLUS and USB_HOST_DATA_MINUS pins from the board lib directly?  It would be very nice to be able to read and write with this library
